tests/dualtor_mgmt/conftest.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from tests.common.helpers.assertions import pytest_require as py_require
+from tests.common.fixtures.ptfhost_utils import change_mac_addresses        # noqa F401
+from tests.common.fixtures.ptfhost_utils import run_garp_service            # noqa F401
+from tests.common.utilities import get_host_visible_vars
+
+
+def pytest_configure(config):
+
+    config.addinivalue_line(
+        "markers", "enable_active_active: mark test to run with 'active_active' ports"
+    )
+
+    config.addinivalue_line(
+        "markers", "skip_active_standby: mark test to skip running with 'active_standby' ports"
+    )
+
+
+@pytest.fixture(scope="module", autouse=True)
+def common_setup_teardown(rand_selected_dut, request, tbinfo, vmhost):
+    # Skip dualtor test cases on unsupported platform
+    supported_platforms = ['broadcom_td3_hwskus', 'broadcom_th2_hwskus', 'cisco_hwskus']
```

Comment:
I suggest checking the `CONFIG_DB` entry instead of hardcoding the platform names.

Reply:
Add a check to ensure the testbed is `dualtor`, and this part is a legacy code, can we put as it is for now?

Reply:
> Please double check the case name in `tests_mark_conditions.yaml`. Renaming the test script may break the rules.

Confirmed that there is no conditional mark related to testcases under `tests/dualtor_mgmt/`